rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc

```diff
@@ -55,8 +64,17 @@ include::modules/rosa-hcp-vpc-manual.adoc[leveloffset=+2]
 * link:https://docs.aws.amazon.com/vpc/latest/userguide/vpc-getting-started.html[Get Started with Amazon VPC]
 * link:https://developer.hashicorp.com/terraform[HashiCorp Terraform documentation]
 
-include::modules/rosa-hcp-byo-oidc.adoc[leveloffset=+1]
-//include::modules/rosa-sts-creating-a-cluster-using-defaults-ocm.adoc[leveloffset=+2]
+[id="rosa-hcp-byo-odic-overview_{context}"]
+== Bring-Your-Own OpenID Connect Configuration
+
+When using a Red-Hat hosted cluster, you can create an OpenID Connect (OIDC) provider that is generated by the CLI and stored within Red Hat's AWS account. This process does not provide a private key for users to access.
```

Comment:
I think it should be OpenID Connect (OIDC) **config**. And another issue is the un-managed OIDC config is not stored in the Red Hat account. @gdbranco Would you help double confirm it too?

> you can create an OpenID Connect (OIDC) provider that is generated by the CLI and stored within Red Hat's AWS account.

Reply:
I think we need some session to properly address both types of Managed vs Unmanaged